pkg/virt-handler/vm.go

```diff
@@ -1584,10 +1586,36 @@ func (d *VirtualMachineController) updateNodeCpuManagerLabel() {
 			log.DefaultLogger().Reason(err).Errorf("failed to set a cpu manager label on host %s", d.host)
 			return
 		}
-		if strings.Contains(string(content), "kubelet") && strings.Contains(string(content), "cpu-manager-policy=static") {
+		if !bytes.Contains(content, []byte("kubelet")) {
```

Comment:
hm I can not connect between PR description and fix, I can see you started to use `bytes` instead of `strings` and I can see you check the kubelet config for the parameter, but how it can fix the test?

Reply:
thanks for the review :) this is not about fixing the test but https://bugzilla.redhat.com/show_bug.cgi?id=1718944, sometimes cpu-manager-policy might be set in the config file instead of kubelet's cmdline, the idea was to check the kubelet config and check the policy file there, however, this approach is bad because `kubelet.conf` doesn't have always the same location. I am going to change solution here and check `cpu_manager_state` files.

/hold

Reply:
thanks for the explanation, it was unclear for me, that we are talking about BZ, in this case, I can see you already got the `kubelet` config path from the `kubelet` cmd line, so it always should point you to the right file, no?

Reply:
yes, it points me to the right file, but before starting virt-handler I would have to use `hostPath` to mount the kubelet.conf, so it's a problem if the file location is not the same, I cannot mount the file later and I cannot assume that the conf file path is static (always the same location).